package.json

```diff
@@ -38,62 +38,29 @@
         "whatwg-fetch": "^3.0.0"
     },
     "scripts": {
-        "prestart": "npm run extract-pot && npm run localize && npm run manifest",
-        "start": "react-scripts start",
-        "lint": "d2-style js check || d2-style text check",
-        "format": "d2-style js apply || d2-style text apply",
+        "start": "d2-app-scripts start",
         "coverage": "npm test -- --coverage",
-        "prebuild": "rm -rf build && mkdir build && npm run manifest && npm run localize",
-        "build": "PUBLIC_URL=. react-scripts build",
-        "build-netlify": "REACT_APP_DHIS2_BASE_URL=\"https://play.dhis2.org/dev\" npm run build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject",
-        "manifest": "d2-manifest package.json ./public/manifest.webapp --manifest.activities.dhis.href=${REACT_APP_DHIS2_BASE_URL:=..}",
-        "extract-pot": "d2-i18n-extract -p src/ -o i18n/",
-        "localize": "d2-i18n-generate -n dashboards-app -p ./i18n/ -o ./src/locales/",
-        "validate-commit": "d2-style js check --staged && d2-style text check --staged && yarn extract-pot && git add ./i18n/",
+        "lint": "d2-style js check || d2-style text check",
+        "build": "d2-app-scripts build",
+        "build:netlify": "yarn build --standalone",
+        "test": "d2-app-scripts test",
         "validate-push": "CI=true yarn test"
     },
-    "manifest.webapp": {
-        "name": "Dashboards app",
-        "description": "DHIS2 core dashboard",
-        "icons": {
-            "48": "icon.png"
-        },
-        "developer": {
-            "url": "",
-            "name": "DHIS2"
-        },
-        "dhis2": {
-            "apiVersion": "34"
-        },
-        "activities": {
-            "dhis": {
-                "href": ".."
-            }
-        }
-    },
-    "eslintConfig": {
-        "extends": "react-app"
-    },
-    "browserslist": [
-        ">0.2%",
-        "not dead",
-        "not ie <= 11",
-        "not op_mini all"
-    ],
     "devDependencies": {
+        "@dhis2/cli-app-scripts": "^4.0.7",
         "@dhis2/cli-style": "^7.0.0",
-        "@dhis2/d2-i18n-extract": "^1.0.8",
-        "@dhis2/d2-i18n-generate": "^1.2.0",
         "d2-manifest": "^1.0.0",
         "enzyme": "^3.11.0",
         "enzyme-adapter-react-16": "^1.15.2",
-        "eslint-config-prettier": "^4.1.0",
-        "eslint-plugin-prettier": "^3.1.3",
-        "eslint-plugin-react-hooks": "^2.3.0",
-        "husky": "^4.2.5",
-        "immutability-helper": "^3.1.1",
-        "prettier": "^1.16.4"
+        "enzyme-to-json": "^3.5.0",
+        "immutability-helper": "^3.1.1"
+    },
+    "jest": {
+        "setupFilesAfterEnv": [
+            "<rootDir>/src/setupTests.js"
+        ],
+        "moduleNameMapper": {
+            "^.+\\.(css|sass|scss)$": "identity-obj-proxy"
+        }
```

Comment:
This should be taken care of automatically by the platform so I don't think it's necessary here?

Reply:
@austin When I remove it then some of the snapshot tests fail with this kind of difference: 

```    -       <div
    -         className="leftControls"
    -       >
    +       <div>
```

Is that what is expected, and I should update the snapshots?